extras/systemd/dynflow-sidekiq@.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=Foreman jobs daemon on sidekiq
+Documentation=https://theforeman.org
+After=network.target remote-fs.target nss-lookup.target
+
+[Service]
+Type=forking
+User=foreman
+TimeoutSec=90
+Environment=RAILS_ENV=production
+Environment=DYNFLOW_SIDEKIQ_SCRIPT=/usr/share/foreman/script/dynflow-sidekiq.rb
+WorkingDirectory=/usr/share/foreman
+ExecStart=/usr/local/bin/sidekiq -e ${RAILS_ENV} -r ${DYNFLOW_SIDEKIQ_SCRIPT} -C /etc/foreman/dynflow/%i.yml
```

Comment:
Does the `%i.yml` imply a separate configuration file is distributed for each worker? or Is that the executor?

Reply:
Some definitions first: The executor in the old deployment was a single process, which did the work and decided what should be done next.
In the new deployment, the executor becomes more of a logical concept than a real thing, but we could say the executor is redis+orchestrator+worker. Another tricky thing here is that orchestrator is just a special worker, which can also act as regular worker, if configured to do so.

Each of the instances (be it orchestrator or some kind of worker) will have its own configuration file. See this commit message for more details on it https://github.com/Dynflow/dynflow/pull/342/commits/2de27040cd40a8fa27597d3776ab84a59b65f03a

Reply:
How are we planning to deploy it? I had assumed a dedicated executor and dedicated workers rather than allowing the executor be a muddled hybrid of the two? I'd be worried the latter is repeating the existing pattern and leading to further confusion from a runtime and support perspective.

Reply:
As per https://github.com/theforeman/foreman/pull/7109#discussion_r337007011 there should be a single orchestrator and a single worker in the default deployment. Merging the orchestrator and the worker is possible, but shouldn't really be done.